packages/http-crawler/src/internals/http-crawler.ts

```diff
@@ -24,6 +24,7 @@ import {
 } from '@crawlee/basic';
 import type { Awaitable, Dictionary } from '@crawlee/types';
 import type { RequestLike, ResponseLike } from 'content-type';
+import * as cheerio from 'cheerio';
```

Comment:
this needs to go to the http crawler dependencies, otherwise it wont work with strict dependency managers like yarn pnp/pnpm.

Reply:
done 🙏
just had a long discussion/rant about pnpm with a friend this weekend 😆

Reply:
you will need to update the lock file too. make sure to have corepack enabled so correct yarn version is used.

Reply:
i'd guess you have yarn linked in your PATH and it overrides what corepack is trying to do

i can do it later myself, no worries.